src/components/Formulários/Formulario.vue

```diff
@@ -0,0 +1,369 @@
+                </p>
+            </div>
+            <div class="form">
+                <form>
+                    <div class="input-box">
+                        <label for="name">Nome Completo</label>
+                        <input type="text" id="name" v-model="name">
+                    </div>
+                    <div class="input-columns">
+                        <div class="input-box email-area">
+                            <label for="email">Email</label>
+                            <input type="text" id="email" v-model="email">
+                            <img class="email-icon" src="@/assets/email.svg" alt="Email Icon">
+                        </div>
+                        <div class="input-box email-area">
+                            <label for="confirmEmail">Confirmar Email</label>
+                            <input type="text" id="confirmEmail" v-model="confirmEmail">
+                            <img class="email-icon" src="@/assets/email.svg" alt="Email Icon">
+                        </div>
+                    </div>
+                    <div class="input-columns">
+                        <div class="input-box">
+                            <label for="cpf">Cpf</label>
+                            <input type="text" id="cpf" v-model="cpf" v-mask="'###.###.###-##'">
+                        </div>
+                        <div class="input-box">
+                            <label for="phone">Celular</label>
+                            <input type="text" id="phone" v-model="phone" v-mask="'(##) #.####-####'">
+                        </div>
+                    </div>
+                    <div class="input-date">
+                        <label for="date">Data de Nascimento</label>
+                        <input type="date" id="date" v-model="birthDate">
+                    </div>
+                </form>
+            </div>
+            <div class="welcome">
+                <p>
+                    Seu cadastro está quase finalizado! Agora, por onde você gostaria que entrassemos em contato para trazer novidades sobre seus investimentos?
+                </p>
+                <div class="checkbox-area">
+                    <input type="checkbox" name="checkEmail" v-model="emailSMS">
+                    <label for="checkEmail"> Email e SMS </label>
+                  </div>
+          
+                  <div class="checkbox-area">
+                    <input type="checkbox" name="checkWhats" v-model="whatsApp">
+                    <label for="checkWhats"> WhatsApp </label>
+                  </div>
+                  <p>
+                    Ao clicar em <strong>continuar</strong> você concorda com a <strong>Politica de Privacidade</strong> e as <strong>Condições de Uso</strong> da <span>Warren</span>. Também que seus dados possam ser utilizados para cadastro, pesquisas e afins.
+                  </p>
+                  <button type="button" @click="nextForm()">Continuar</button>
```

Comment:
Se não está sendo passado nenhum dado pode remover os parênteses ao chamar o método.
```suggestion
                  <button type="button" @click="nextForm">Continuar</button>
```